e2e/cypress/integration/account_settings/display/message_display_mode_spec.js

```diff
@@ -12,55 +12,53 @@
 
 import {getRandomId} from '../../../utils';
 
-describe('Account Settings > Display > Message Display', () => {
+describe('Messaging', () => {
```

Comment:
Maybe either move this file under `Messaging` folder or change describe to `Account Settings` and update zephyr folder to point to `Account Settings`

Reply:
Good catch. Thanks Joseph!